src/diffusers/utils/peft_utils.py

```diff
@@ -184,7 +186,7 @@ def set_weights_and_activate_adapters(model, adapter_names, weights):
                     module.set_adapter(adapter_name)
                 else:
                     module.active_adapter = adapter_name
-                module.scale_layer(weight)
+                module.set_scale(adapter_name, weight)
```

Comment:
What happens if `adapter_name` is not available for this lora layer? Is this handled in PEFT to do nothing if adapter is unavailable?

Reply:
I think it errors it out from `peft` in case that is hit. @younesbelkada can confirm. 

From what I recollect, the users are required to provide `adapter_name` when dealing with multiple LoRAs otherwise, for the single case, the default naming is used.

Reply:
https://github.com/huggingface/peft/pull/1034 should take care of that

Reply:
https://github.com/huggingface/peft/pull/1034 is merged. So, no blockers.